src/zcl_abapgit_repo.clas.abap

```diff
@@ -107,15 +106,27 @@ CLASS zcl_abapgit_repo DEFINITION
         VALUE(rt_list) TYPE scit_alvlist
       RAISING
         zcx_abapgit_exception .
-
+    METHODS has_remote_source
+    ABSTRACT
+      RETURNING
+        VALUE(rv_yes) TYPE abap_bool .
+    METHODS status
+      IMPORTING
+        !io_log TYPE REF TO zcl_abapgit_log OPTIONAL
+      RETURNING
+        VALUE(rt_results) TYPE zif_abapgit_definitions=>ty_results_tt
+      RAISING
+        zcx_abapgit_exception .
   PROTECTED SECTION.
 
     DATA mt_local TYPE zif_abapgit_definitions=>ty_files_item_tt .
     DATA mt_remote TYPE zif_abapgit_definitions=>ty_files_tt .
-    DATA mv_do_local_refresh TYPE abap_bool .
+    DATA mv_request_local_refresh TYPE abap_bool .
     DATA mv_last_serialization TYPE timestamp .
     DATA ms_data TYPE zif_abapgit_persistence=>ty_repo .
     DATA mv_code_inspector_successful TYPE abap_bool .
+    DATA mv_remote_is_initialized TYPE abap_bool .
```

Comment:
I see what you mean ... let's discuss, this topic is not that clear imho

1) Ideally (ideally) I would remove both attributes ...
2) `mv_request_local_refresh` is used only in `get_files_local` - do you think checking `mt_local is initial` is a safe replacement ?
3) same for `mv_remote_is_initialized`, in fact just used in `fetch_remote`. Would the `mt_remote is initial` be safe enough ?

What do you think ?

If both "no", I'll rename `mv_remote_is_initialized` to `mv_request_remote_refresh`. Because the mv_request_LOCAL_refresh fits it's role really well now.

Reply:
I would also like to remove both attributes, but I'm not sure it is safe, I think "no" for this PR, we can move the issue to a new issue/PR

Suggest just renaming so both are similar

Reply:
ok, done